frontend/rust-lib/flowy-ai/src/local_ai/watch.rs

```diff
@@ -5,13 +5,15 @@ use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver};
 use tracing::{error, trace};
 
 #[cfg(any(target_os = "windows", target_os = "macos", target_os = "linux"))]
+#[allow(dead_code)]
 pub struct WatchContext {
   #[allow(dead_code)]
```

Comment:
Do we still need to keep `#[allow(dead_code)]` for the `watcher`?

Reply:
we dont need this anymore, I'll remove it